consensus/polybft/state_store_governance.go

```diff
@@ -0,0 +1,183 @@
+package polybft
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/contractsapi"
+	"github.com/0xPolygon/polygon-edge/helper/common"
+	bolt "go.etcd.io/bbolt"
+)
+
+var (
+	governanceEventsBucket              = []byte("governanceEvents")
+	clientConfigBucket                  = []byte("clientConfig")
+	clientConfigKey                     = []byte("clientConfigKey")
+	lastProcessedGovernanceEventsBucket = []byte("lastProcessedGovernanceEvents")
+	lastProcessedGovernanceBlockKey     = []byte("lastProcessedGovernanceBlockKey")
+
+	errNoLastProcessedGovernanceBlock = errors.New("no last processed block for governance in db")
+	errClientConfigNotFound           = errors.New("client (polybft) config not found in db")
+)
+
+type eventsRaw [][]byte
+
+// Bolt db schema:
+//
+// governance events/
+// |--> epoch -> slice of contractsapi.EventAbi
```

Comment:
Add last processed governance events and client config to schema as well?

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1749/commits/6bcffbbd344e5db00b863d22c1872ace2a23a9f6